### ISSUE TITLE
Outdated Interaction (usage of Node) documentation on JuliaHub.

### ISSUE THREAD

**Issue by pitipatw** [MakieOrg/Makie.jl#4011]
I'm working on some interaction in Julia, and found a documentation for "Julia-Makie-Interaction" which is still using `Node`. 

https://docs.juliahub.com/MakieGallery/Ql23q/0.2.17/interaction.html

Im not sure if this should be kept there or remove completely since `Node` is depreciated.

**Comment by jkrumbiegel** [comment]
MakieGallery has been archived since 2021, so all that information is quite outdated. I understand that it can be confusing to find such information while googling. Maybe this one could have a deprecation notice added to the docs?

**Comment by pitipatw** [comment]
> MakieGallery has been archived since 2021, so all that information is quite outdated. I understand that it can be confusing to find such information while googling. Maybe this one could have a deprecation notice added to the docs?

I totally agree with you. Marking "deprecation" would make it a lot clearer than the current document.

Is the moderator on MakieGallery the same as on Github?
